web/src/routes/categories/components/CategoryList.js

```diff
@@ -42,13 +42,17 @@ type PropsType = {|
  */
 class CategoryList extends React.PureComponent<PropsType> {
   render () {
-    const { categories, title, thumbnail, content, query, onInternalLinkClick } = this.props
+    const {
+      categories, query, onInternalLinkClick, formatter, category
+    } = this.props
     return (
       <div>
-        {thumbnail && <Centering><CategoryIcon src={thumbnail} alt='' /></Centering>}
-        {title && <Caption title={title} />}
-        {content && <RemoteContent dangerouslySetInnerHTML={{ __html: content }}
+        {category && category.thumbnail && <Centering><CategoryIcon src={category.thumbnail} alt='' /></Centering>}
```

Comment:
Alternatively you could also do which is cleaner imo, same for other checks:
```suggestion
        {category?.thumbnail && <Centering><CategoryIcon src={category.thumbnail} alt='' /></Centering>}
```
See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining for more infos.